mmv1/third_party/terraform/services/logging/resource_logging_project_sink.go

```diff
@@ -39,6 +39,12 @@ func ResourceLoggingProjectSink() *schema.Resource {
 		Default:     false,
 		Description: `Whether or not to create a unique identity associated with this sink. If false (the default), then the writer_identity used is serviceAccount:cloud-logs@system.gserviceaccount.com. If true, then a unique service account is created and used for this sink. If you wish to publish logs across projects, you must set unique_writer_identity to true.`,
 	}
+	schm.Schema["custom_writer_identity"] = &schema.Schema{
```

Comment:
Just want to double-check that this & unique_writer_identity are specific to the _project_ logging sink and shouldn't be supported on the org & folder sinks?

Reply:
`unique_writer_identity` is not project specific.

Reply:
thanks, what about `custom_writer_identity`?

Reply:
It's not project specific either.:)
We will have following up prs to add them to fold/org level sinks. :)

Reply:
ack, happy to leave that for separate PRs.